openhands/core/config/config_utils.py

```diff
@@ -0,0 +1,44 @@
+from enum import Enum
+from types import UnionType
+from typing import get_args, get_origin
+
+OH_DEFAULT_AGENT = 'CodeActAgent'
+OH_MAX_ITERATIONS = 100
+
+
+class UndefinedString(str, Enum):
+    UNDEFINED = 'UNDEFINED'
+
+
+def get_field_info(f):
```

Comment:
This can be moved to config_functions.py, or rather, all functions can be in a file named simply utils.py.

Reply:
Will do, thanks